corehq/apps/userreports/tests/test_transforms.py

```diff
@@ -71,8 +65,27 @@ def setUp(self):
     (None, ''),
 ), TestEthiopianConversion)
 def test_ethiopian_to_gregorian(self, date_string, expected_result):
-    self.assertEqual(expected_result, self.transform(date_string))
+    transform = TransformFactory.get_transform({
+        "type": "custom",
+        "custom_type": "ethiopian_date_to_gregorian_date",
+    }).get_transform_function()
+    self.assertEqual(expected_result, transform(date_string))
+
+
+@generate_cases((
+    ('2017-05-19 ', date(2009, 9, 11)),
+    ('2017-09-16 ', date(2010, 1, 6)),
```

Comment:
This isn't the inverse of the corresponding test in test_ethiopian_to_gregorian, and I'm not totally sure why, possibly because date() can't handle 13 months? I tried using an online converter and it's still unclear

<img width="837" alt="screen shot 2018-01-26 at 10 49 54 am" src="https://user-images.githubusercontent.com/35747290/35447817-ab20bd68-0286-11e8-9238-d39b0b1b48f7.png">
<img width="847" alt="screen shot 2018-01-26 at 10 49 48 am" src="https://user-images.githubusercontent.com/35747290/35447818-ab2eb38c-0286-11e8-942d-b4ebc85a9553.png">